src/main/client/src/components/@commons/Basics.tsx

```diff
@@ -1,10 +1,47 @@
 import React from 'react'
-import { Text } from '@chakra-ui/react'
+import { Button, ButtonProps, Text, LinkProps as ChakraLinkProps, Link } from '@chakra-ui/react'
+import { Link as RouterLink } from 'react-router-dom'
+import { LinkIcon } from '@chakra-ui/icons'
 
 export const Paragraph: React.FC = ({ children, ...props }) => {
```

Comment:
create a `Paragraph.tsx` for this.

Reply:
This will happen in the future, but some people might use it now. Will create when they are merged.